src/core/js/drawer.js

```diff
@@ -6,6 +6,10 @@ define([
     var DrawerCollection = new Backbone.Collection();
     var Drawer = {};
 
+    DrawerCollection.comparator = function (item) {
```

Comment:
This can be shortened to:
```js
DrawerCollection.comparator = 'drawerOrder';
```
or, even more neatly, just passed in as an option up on line 6:
```js
var DrawerCollection = new Backbone.Collection(null, { comparator: 'drawerOrder' });
```